tests/integration/conftest.py

```diff
@@ -163,7 +165,35 @@ def protostar_fixture(
     session_mocker: MockerFixture,
     protostar_project_root_path: Path,
 ) -> ProtostarFixture:
+    """@deprecated: Use `create_protostar_project` fixture instead."""
     return build_protostar_fixture(
         mocker=session_mocker,
         project_root_path=protostar_project_root_path,
     )
+
+
+class CreateProtostarProjectFixture(Protocol):
```

Comment:
I don't understand how this is helping us, is that something that we really need?  Typing the fixtures seems a little excessive

Reply:
it helps IDE autocompletion

Reply:
And refactoring, and gives immediate feedback in case of misuse, and [insert type checking benefits]